src/components/BookingModal/ModalStatusBanner/index.jsx

```diff
@@ -70,7 +71,8 @@ class ModalStatusBanner extends Component {
           </div>
         )}
         <div>
-          {!isCancelled && (
+          {!isCancelled &&
+            !isApproved && (
```

Comment:
this is cleaner on one line `{!isCancelled && !isApproved && (`

Reply:
@d1wilko I think eLint is saving it that way

Reply:
weird... but ok